ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.5.yaml

```diff
@@ -98,13 +98,20 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: gcp-ipi-proxy-f30
+- as: aws-ipi-f30
   cron: 5 4 24 * *
   steps:
-    cluster_profile: gcp-qe
+    cluster_profile: aws-qe
     env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      LOKI_ENABLED: "false"
       UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.14 and @gcp-ipi and not @fips'
       UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.14 and @gcp-ipi and not @fips'
+    post:
```

Comment:
Since we switched the platform, the tags need update, too.

Reply:
ah, good catch!